test/e2e/setup.go

```diff
@@ -207,23 +207,6 @@ func setup(ctx context.Context) error {
 	return nil
 }
 
-func done(ctx context.Context) error {
```

Comment:
It looks like removing this will impact local E2E testing (all clusters provisioned outside the pipeline will exist until they are purged). Is this desirable? I think I'm fine with it either way, but worth a discussion. This may impact our cloud spend.

Reply:
> all clusters provisioned outside the pipeline will exist until they are purged

that behavior doesn't change right? local dev clusters dont delete when you run a local e2e anyways. clusters were only deleted if we are in CI and delete cluster is not false
https://github.com/Azure/ARO-RP/pull/1955/files#diff-32dd91f1588cf2a777ac4a9002aa92e6c9ed64cb49d0b53d0efe4a8deefaf030L212

Reply:
The behavior will be the same as it is now, but https://msazure.visualstudio.com/AzureRedHatOpenShift/_wiki/wikis/ARO.wiki/166757/Debug-E2E will need to be updated with the "new" way of retaining an e2e cluster (probably just commenting out the `cleanup` stage from the pipeline)